client/routes/app/home/Home.html

```diff
@@ -0,0 +1,41 @@
+<h1>This is the home page</h1>
+
+<h2><a href="{{ asr.makePath('app.respirator-picker') }}">Pick your respirator!</a></h2>
+
+<h2>Content:</h2>
+<ul>
+	{{#each navigation as item}}
+
+	<li>
+	
+		{{#if item.children}}
+			
+		<Category title="{{item.title}}">
```

Comment:
It looks like you could probably just use the `Accordion` component here?

Reply:
Probably.  I duplicated the component because I expect it to diverge from `Accordion` once I begin styling more.  I may unify them again if I discover that they really are semantically identical.

Reply:
If it's just styling, you could get away with passing a `className` into the `Accordion` and then using different stylesheets for the different contexts.